buildSrc/src/main/java/org/elasticsearch/gradle/precommit/TestingConventionsTasks.java

```diff
@@ -323,9 +319,11 @@ private boolean isAnnotated(Class<?> clazz, Method method, Class<?> annotation)
     }
 
     private FileCollection getTestsClassPath() {
-        // This is doesn't need to be annotated with @Classpath because we only really care about the test source set
+        // Loading the classes does depend on the @Classpath but we can have better caching if we don't annotate this
+        // as such as the conventions as such don't depend on the classpath and any missing classes would be detected
```

Comment:
This sentence doesn't make grammatical sense...I can't figure out what was intended either.